v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md

```diff
@@ -265,36 +265,39 @@ Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负
 
 ### 查看 TiDB 慢查询日志
```

Comment:
Please also update these changes in dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md.